app/src/pages/OnDeviceDisplay/ProtocolSetup.tsx

```diff
@@ -408,3 +427,29 @@ export function ProtocolSetup(): JSX.Element {
     </Flex>
   )
 }
+
+interface ProtocolSetupSkeletonProps {
+  cancelAndClose: () => void
+}
+function ProtocolSetupSkeleton(props: ProtocolSetupSkeletonProps): JSX.Element {
+  return (
+    <Flex flexDirection={DIRECTION_COLUMN} gridGap={SPACING.spacingXXL}>
+      <Flex justifyContent={JUSTIFY_SPACE_BETWEEN}>
+        <Flex flexDirection={DIRECTION_COLUMN} gridGap="0.25rem">
+          <Skeleton height="2rem" width="7rem" backgroundSize="64rem" />
+          <Skeleton height="2rem" width="28rem" backgroundSize="64rem" />
+        </Flex>
+        <Flex gridGap={SPACING.spacing5}>
+          <CloseButton onClose={() => props.cancelAndClose()} />
```

Comment:
For consistency, probably we would like avoid `props.`

```ts
function ProtocolSetupSkeleton({cancelAndClose}: ProtocolSetupSkeletonProps): JSX.Element {
or
const { cancelAndClose } = props
```

```suggestion
          <CloseButton onClose={() => cancelAndClose()} />
```